File: server/Models/Utakmica.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;
using System.Text.Json.Serialization;
using System;

namespace Models
{

    [Table("Utakmica")]
    public class Utakmica
    {
        [Key]
        public int MecID { get; set; }

        [Required]
        public Sezona Sezona { get; set; }

        [Required]
        public int Kolo { get; set; }
        
        [Required]
        public Klub Domacin { get; set; }

        [Required]
        public int golovi_domacin { get; set; }
        
        [Required]
        public Klub Gost { get; set; }

        [Required]
        public int golovi_gost { get; set; }

        [Required]
        public Sudija sudija { get; set; }
    }
}

File: client/Utakmica.js
export class Utakmica {

    constructor(domacin, golovi_domacin, gost, golovi_gost, sezona, kolo,sudija) {
        this.domacin = domacin;
        this.golovi_domacin = golovi_domacin;
        this.gost = gost;
        this.golovi_gost = golovi_gost;
        this.sezona = sezona;
        this.kolo = kolo;
        this.sudija = sudija;
    }

    crtajUtakmicu(table) {

        var tr = document.createElement("tr");
        table.appendChild(tr);

        var el = document.createElement("td");
        el.innerHTML = this.kolo;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.domacin.naziv;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.golovi_domacin;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.golovi_gost;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.gost.naziv;
        el.nodeValue = this.gost.naziv;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.sudija.ime + " " + this.sudija.prezime;
        tr.appendChild(el);
    }


    
}